vpdq/python/README.md

```diff
@@ -0,0 +1,45 @@
+# vpdq
+This is a Python binding library to the vPDQ (video PDQ) hashing algorithm by Facebook. The project is at [github](https://github.com/facebook/ThreatExchange/tree/main/vpdq)
+
+## Installation
+### Install library
+```
+pip install --upgrade pip
+pip install vpdq
```

Comment:
(ignorable): Suggest only showing the installation command itself, pip will complain it needs an update when you use it